### PR TITLE
Clarify the connection settings for duckdb and lakeFS

### PR DESCRIPTION
**Pull request by talSofer**

### CODE REVIEW

File: docs/integrations/duckdb.md [hunk 1]
```diff
@@ -26,17 +26,15 @@ Then run the following to configure the connection.
...
+SET s3_access_key_id='AKIAIOSFODNN7EXAMPLE'; -- the access credentials for your lakeFS user
+SET s3_secret_access_key='wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY'; -- the access credentials for your lakeFS user
 SET s3_url_style='path';
 
 -- Uncomment in case the endpoint listen on non-secure, for example running lakeFS locally.
 -- SET s3_use_ssl=false;
 ```
 
-* `s3_endpoint` is the host (and port, if necessary) of your lakeFS server
-* `s3_access_key_id` and `s3_secret_access_key` are the access credentials for your lakeFS user
```
Comment (nopcoder) [inline docs/integrations/duckdb.md:39]: all or nothing? suggest we keep the legend or have all the field's descriptions inside the snippet.

File: docs/integrations/duckdb.md [hunk 2]
```diff
@@ -26,17 +26,15 @@ Then run the following to configure the connection.
 
 ```sql
 SET s3_region='us-east-1';
-SET s3_endpoint='lakefs.example.com';
-SET s3_access_key_id='AKIAIOSFODNN7EXAMPLE';
-SET s3_secret_access_key='wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY';
+SET s3_endpoint='lakefs.example.com'; -- the host (and port, if necessary) of your lakeFS server
+SET s3_access_key_id='AKIAIOSFODNN7EXAMPLE'; -- the access credentials for your lakeFS user
+SET s3_secret_access_key='wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY'; -- the access credentials for your lakeFS user
```
Comment (nopcoder) [inline docs/integrations/duckdb.md:31]: Suggest to explain each one above the field - like we did for s3_use_ssl.

**Review by nopcoder — CHANGES_REQUESTED** [review]

File: docs/integrations/duckdb.md (hunk 1 above)
Comment (talSofer) [reply docs/integrations/duckdb.md:39]: done, I removed the line about path style, which seems redundant

File: docs/integrations/duckdb.md [hunk 3]
```diff
@@ -25,21 +25,20 @@ LOAD httpfs;
 Then run the following to configure the connection. 
 
 ```sql
+-- `s3_region` is the S3 region on which your bucket resides. If local storage, or not S3, then just set it to `us-east-1`.
```
Comment (nopcoder) [inline docs/integrations/duckdb.md:28]: The use of ` in side `sql` block will not render as markdown - suggest to use single or double quotes.

**Review by nopcoder — CHANGES_REQUESTED** [review]
LGTM

File: docs/integrations/duckdb.md (hunk 3 above)
Comment (talSofer) [reply docs/integrations/duckdb.md]: done

**Review by nopcoder — APPROVED** [review]

**Pull request merged**